adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -378,132 +369,131 @@ class ADAMContext(val sc: SparkContext) extends Serializable with Logging {
       ContextUtil.getConfiguration(job))
     if (Metrics.isRecording) records.instrument() else records
 
-    applyPredicate(records.map(p => vcc.convertToAnnotation(p._2.get)), predicate)
+    records.map(p => vcc.convertToAnnotation(p._2.get))
   }
 
-  def loadParquetVariantAnnotations[U <: ADAMPredicate[DatabaseVariantAnnotation]](
+  def loadParquetVariantAnnotations(
     filePath: String,
-    predicate: Option[Class[U]] = None,
+    predicate: Option[FilterPredicate] = None,
     projection: Option[Schema] = None): RDD[DatabaseVariantAnnotation] = {
-    adamLoad[DatabaseVariantAnnotation, U](filePath, predicate, projection)
+    adamLoad[DatabaseVariantAnnotation](filePath, predicate, projection)
   }
 
-  def loadVariantAnnotations[U <: ADAMPredicate[DatabaseVariantAnnotation]](
+  def loadVariantAnnotations(
     filePath: String,
-    predicate: Option[Class[U]] = None,
+    predicate: Option[FilterPredicate] = None,
```

Comment:
Looks like these methods are mostly used for the `Adam2*` functions anyway. Agree it makes sense to just remove predicates altogether since they aren't being used (or useful).